console-subscriber/src/visitors.rs

```diff
@@ -1,4 +1,4 @@
-//! These visitors are respondible for extracing the relevan
+//! These visitors are respondible for extracting the relevan
```

Comment:
probably should also fix these:
````suggestion
//! These visitors are responsible for extracting the relevant
```